Marlin/temperature.cpp

```diff
@@ -1544,70 +1546,79 @@ void Temperature::isr() {
   #endif
 
   #if DISABLED(SLOW_PWM_HEATERS)
```

Comment:
I have indeed. The `ENABLED` macro uses preprocessor voodoo that only works in preprocessor expressions.